experimental/relativetime/tests/tests.rs

```diff
@@ -0,0 +1,798 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use fixed_decimal::FixedDecimal;
+use icu_locid::locale;
+use icu_relativetime::{options::Numeric, RelativeTimeFormatter, RelativeTimeFormatterOptions};
+use writeable::assert_writeable_eq;
+
+macro_rules! generate_test {
+    ($test_name: ident, $constructor: ident, $options: expr, [$(($time: literal, $expected: literal)),+ $(,)?]) => {
+        #[test]
+        fn $test_name(){
+            let relative_time_formatter = RelativeTimeFormatter::$constructor(
+                &icu_testdata::unstable(),
+                &locale!("ar").into(),
```

Comment:
Please use English so we can sanity check that there are no mixups anywhere.

Reply:
Added tests containing English locale too. "ar" locale had cases not covered by "en" hence using both.